Tasks/Src/RTOS_BPS.c

```diff
@@ -83,19 +83,17 @@ void RTOS_BPS_MutexPost(BPS_OS_MUTEX* mutex, BPS_OS_OPT options) {
  * @return  nothing to see here
  */
 void RTOS_BPS_TaskCreate(
-    OS_TCB              *p_tcb,
-    BPS_OS_TASK_NAME    *p_name,
-    BPS_OS_TASK_PTR      p_task,
-    void                *p_arg,
-    BPS_OS_TASK_PRIO     prio,
-    CPU_STK             *p_stk_base,
-    BPS_OS_TASK_STK_SIZE stk_size
+    BPS_OS_TCB      *p_tcb,
```

Comment:
Nice yeah, these look much better. This should probably work but we'll need to start doing comprehensive tests for the wrapper functions eventually.